apps/bots/.env.example

```
@@ -20,7 +20,13 @@ KAPSO_API_KEY=                          # From Kapso dashboard
 KAPSO_PHONE_NUMBER_ID=                  # From: kapso whatsapp numbers resolve --phone-number "<number>" --output json
 WHATSAPP_PHONE_NUMBER=                  # E.164 number without +, e.g. 15551234567 (used for wa.me links)
 KAPSO_WEBHOOK_SECRET=                   # Webhook signature secret (set when creating phone-number webhook)
-WHATSAPP_WEBHOOK_PORT=3001              # Port the webhook HTTP server listens on
+
+# ── Bot Server ──
+# Each bot starts an HTTP server on this port with GET /health for uptime
+# monitoring (BetterStack, Docker healthchecks). WhatsApp also serves its
+# Kapso webhook (POST /webhook) on the same server.
+# Set per-container via Docker Compose environment overrides.
+BOT_SERVER_PORT=                         # e.g. 3200 (discord), 3201 (slack), 3202 (telegram), 3001 (whatsapp)
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)